zwad/download_feature_set.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
```

Comment:
We use `#!/usr/bin/env python` here in other files.

Reply:
https://github.com/snad-space/zwad/blob/9204d31aedd3878892c46ec0b248da870919491d/zwad/nn.py#L1

Reply:
https://github.com/snad-space/zwad/blob/9204d31aedd3878892c46ec0b248da870919491d/zwad/aad.py#L1

Which one is the right?